trillian/integration/ct_hammer/main.go

```diff
@@ -270,7 +271,8 @@ func main() {
 	var wg sync.WaitGroup
 	for _, c := range cfg {
 		wg.Add(1)
-		pool, err := integration.NewRandomPool(*httpServers, c.PublicKey, c.Prefix)
+		auth := fmt.Sprintf("Bearer %s", *bearerToken)
+		pool, err := integration.NewRandomPool(*httpServers, c.PublicKey, c.Prefix, auth)
```

Comment:
The `len(*bearerToken)` check is missing. If the flag `bearer_token` is not provided, the `auth` value will be `Bearer ` and will be passed into `NewRandomPool`.

Reply:
Oh yeah good spot, thanks.